docs/how_to/data_visualization/views.md

```diff
@@ -0,0 +1,66 @@
+# How to define views
+
+:::{admonition} What does this guide solve?
+:class: important
+This guide will go through how to define views on your dashboard
+:::
+
+## Defining views
+A view is the final output of a dashboard.
+Views are located in the target area and can be a table or numerous kinds of plots by utilizing the [Holoviz ecosystem](https://holoviz.org/).
```

Comment:
```suggestion
Views are located in the `target` area and can take the form of various visual components. Lumen includes many [view types](../../reference/view/index.md), and is built so that you can easily use components from the [Holoviz ecosystem](https://holoviz.org/), such as a [scatter plot from hvPlot](https://hvplot.holoviz.org/reference/pandas/scatter.html), or an [indicator from Panel](https://panel.holoviz.org/reference/index.html#indicators).
```